### ISSUE TITLE
[New DL] Page breadcrumb is missing an accessible text label

### ISSUE THREAD

**Issue by francinen** [Shopify/polaris-react#3271]
## Issue summary
When the new design language is enabled, the breadcrumb content passed to the `Page` component doesn't get rendered in the DOM. This raises an accessibility error when running automated tests and is a regression from the previous design language.

### With new design language
![image](https://user-images.githubusercontent.com/8534230/93611570-77e60880-f99c-11ea-8387-9fe88923230c.png)

### With previous design language
![image](https://user-images.githubusercontent.com/8534230/93611524-6997ec80-f99c-11ea-8572-7d3b01be3fb2.png)

## Expected behavior
Even if the text label is not visually rendered on the page, it should at least be rendered as a visually hidden label or aria-label. 

## Actual behavior
We're not rendering either a visible or visually hidden label for the breadcrumb component.

![image](https://user-images.githubusercontent.com/8534230/93611512-66046580-f99c-11ea-8a11-f51d14c61b5c.png)

## Steps to reproduce the problem

1. Navigate to a page that includes a breadcrumb link in the Page header
2. Enable the new design language
3. Inspect the breadcrumb component. No text label is rendered for the link.

## Specifications

- Are you using the React components? (Y/N):
- Polaris version number:
- Browser:
- Device:
- Operating System:

Or run `npx envinfo --system --binaries --browsers --npmPackages react,react-dom,@shopify/polaris` to provide specifications on your environment including version numbers, browser, device, and operating system.

Paste the results here:

```bash

```
